frontend/src/context/SessionContext.tsx

```diff
@@ -25,11 +29,11 @@ export const SessionProvider: FC = ({ children }) => {
   const [, setCookie, removeCookie] = useCookies(['token']);
   const history = useHistory();
   const location = useLocation();
-
-  const [queryMe, { loading, data, client }] = useLazyQuery(ME_QUERY, {
+  const [login] = useMutation(LOGIN_MUTATION);
+  const [register] = useMutation(REGISTER_MUTATION);
+  const [queryMe, { data, client }] = useLazyQuery(ME_QUERY, {
```

Comment:
Can you add type for `useMutation` and `useLazyQuery`
Ref: [https://www.apollographql.com/docs/react/development-testing/static-typing/#usemutation]( https://www.apollographql.com/docs/react/development-testing/static-typing/#usemutation)